### DISCUSSION TITLE
failed to call API: error sending request for url

### DISCUSSION THREAD

**Discussion by AFoolshfish** [nextai-translator/nextai-translator#1157] [Q&A]
客户端无法使用，显示"failed to call API: error sending request for url (https://api.openai.com/v1/chat/completions): error trying to connect: tcp connect error: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 (os error 10060)"
![image](https://github.com/openai-translator/openai-translator/assets/153589030/ff0519a1-a8ce-41a3-86bf-149cb261b601)

已知API可正常使用，网络也没问题，求教如何解决这个问题

**Comment by wuchunhua** [comment]
请问你解决了么，我这边也是这个问题

**Comment by yetone** [comment]
请检查防火墙或者翻墙软件

**Comment by wzwmao** [comment]
"failed to call API: error sending request for url (http://127.0.0.1:11434/v1/chat/completions): error trying to connect: tcp connect error: 由于目标计算机积极拒绝，无法连接。 (os error 10061)",不知道怎么解决

**Comment by demoliisher** [comment]
My solution is restart the translator after you change the configuration. Remember to exit fully at first
